### ISSUE TITLE
EDIT UI - parking radius sizes

### ISSUE THREAD

**Issue by wkitty42** [Portree-Kid/flightgear-airports#34]
while i appreciate the ease with using A, B, C, D, etc for the parking radius, i need to see the numbers...

eg: terminal parking that can accept radius 14 and 16... but if eg: C covers 14 - 18, then possibly too large craft will be allowed... i would like to see the actual numbers, please... if there's some document somewhere covering the letters to radius numbers, i'm happy to be pointed to it, too...

JFYI: i ran a search on all the traffic files for the radiuses... these are the stats i came up with...
```
~/flightgear-dev/release/install/flightgear/data/AI/Traffic$ find . -type d \( -path .git -o -path .svn -o -path .cvs \) -prune -o -type f -iname "*.xml" -exec grep -EHin -e "radius\>" {} \; | cut -d">" -f2 | cut -d"<" -f1 | sort | uniq -c
    509 10
    397 11
    526 12
      1 127
    756 13
   2200 14
    102 15
    394 16
   4225 17
   1509 18
    609 19
      8 2
     10 22
     58 23
    524 24
     49 26
   2505 30
    579 32
     54 34
    197 40
     22 6
      2 7
      5 8
      7 9
```
the first column is the number of occurrences... the second number is the radius size... yes, i've already tracked that one wild one down and will be reporting it on the forums soon-ish...

**Comment by BecOzIcan** [comment]
http://wiki.flightgear.org/Aircraft_radii

**Comment by gooneybird47** [comment]
I second this. While this rightly centred on commercial traffic you still have to use Taxidraw to add GA or military traffic parking positions so being able to enter the numbers directly would be great.

Previously I've been using the following for GA types.

8   Cessna 150/172, PA-28, Mooney

10  Beech 200

14  Bombardier Global

2   Helipads - although I'm doing less of these lately due other people having a habit of plonking a static helicopter on the only helipad at many airports.

**Comment by BecOzIcan** [comment]
I don't think Keith would have any issue in replacing the copy on his buttons from "D" to '26 (D)" for example 

All of the aircrafts you listed can be accommodated by our existing gate sizes : 7.5 (A) or 14 (B) aside from the Beech. 

Now I am failing to see the use case for an extra size (10) : That would be a situation where you want to pack more GA gates in a single space (using 14 would be wasting space) but GA traffic is handled at local/user level only so it would benefit at best a handful of users ?

For helipads, I am sending you a direct email. No extra size required here

**Comment by gooneybird47** [comment]
I use a 10 gate size as many of the twin props are to big for the 7.5 (A) gate. Many airports have ramps for this sized "feeder" aircraft but will understand if you don't want to over complicate things.

**Comment by BecOzIcan** [comment]
I just need to look at impact on process and automated tests if adding a new radii.  Can we park this one (hahaha) and treat as 'future improvement' for the time being? will revert.

**Comment by wkitty42** [comment]
> http://wiki.flightgear.org/Aircraft_radii

thanks for this...

**Comment by wkitty42** [comment]
> I don't think Keith would have any issue in replacing the copy on his buttons from "D" to '26 (D)" for example

i hope he doesn't... it is a good solution...

> All of the aircrafts you listed can be accommodated by our existing gate sizes : 7.5 (A) or 14 (B) aside from the Beech.

i think i was using the wingspan to determine the GA parking spaces i've been doing but i don't recall for sure... this is especially for separate GA parking areas for the single engine props compared to the dual engine props and those from the small GA jets...

> Now I am failing to see the use case for an extra size (10) : That would be a situation where you want to pack more GA gates in a single space (using 14 would be wasting space) but GA traffic is handled at local/user level only so it would benefit at best a handful of users ?

specifically about "handled at a local/user level only"... i'm planning on adding my current GA flight traffic file(s) to fgdata at some point... my work on adding GA parking to airports in my area and/or the areas i fly, is for those craft... currently it would benefit those that fly in the areas i do GA for but the hope is eventually there will be more GA in some areas where they are common and everyone can enjoy them...

but yes, using 10 keeps larger craft out of those spots reserved for the smaller ones that size and less... and it especially lets us pack more craft into a zone like can be seen in satellite photos... i do try to be fairly accurate when i can :cat:

**Comment by Portree-Kid** [comment]
Is there consensus? 

1. Add radii to the letters
2. Add 10

**Comment by BecOzIcan** [comment]
Yes.

**Comment by BecOzIcan** [comment]
Please do not assign any letter to it. Just "10" given this is not a standardized size

**Comment by gooneybird47** [comment]
Works for me.

**Comment by LGBudd** [comment]
> specifically about "handled at a local/user level only"... i'm planning on adding my current GA flight traffic file(s) to fgdata at some point

I live in an area where air traffic is mostly GA aircraft with occasional larger aircraft (3-4 times a day), so I've developed some GA traffic for the area. In reality, much of U.S. midwest traffic actually consists, in large part, of GA Traffic. So I agree with @wkitty42 and I, too, would like to get some GA Traffic incorporated into FGData as well. Of course, there is no real "schedule" to much of this, as they are private aircraft, either businesses, emergency airlifts, or private planes. So I've been trying to think of a way to randomly generate such traffic for any area the user would like. I don't know enough about nasal or python (at least not yet) to know whether it could be done using those languages. It's something to think about and maybe discuss, however. In the meantime, however, it would be nice to have the ability to add some GA Traffic to FGData.

Regarding the "standard" radius sizes, I'm glad to see there is a "7.5" and, in the future, a "`10" size to accommodate the smaller aircraft. I've often run across many "6" radius parking positions together. That is the only size I've found that will fit into the space given by the markings on the ground and it is confirmed by what I see at the local airport. I guess we can get by without adding all of these very small parking positions to an airport as a size 7.5, but it would be nice to have the ability to add them all to keep things as realistic as possible.

**Comment by BecOzIcan** [comment]
"it would be nice to have the ability to add some GA Traffic to FGData." : I believe AI scenarios have been developed exactly for that.

**Comment by wkitty42** [comment]
but AI Scenarios are not the same thing as AI Traffic... i can do a lot of stuff with AI Traffic and nothing at all with AI Scenarios... FWIW...

**Comment by Portree-Kid** [comment]
Fixed in 0.0.18
